reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md

```diff
@@ -48,67 +48,83 @@ Rename-Item -Path "c:\logfiles\daily_file.txt" -NewName "monday_file.txt"
 
 ### Example 2: Rename and move an item
 
-This example shows that you cannot use `Rename-Item` to both rename and move an item.
-Specifically, you cannot supply a path for the value of the **NewName** parameter, unless the path is identical to the path specified in the **Path** parameter.
-Otherwise, only a new name is permitted.
+You can't use `Rename-Item` to both rename and move an item. Specifically, you can't supply a path
+for the value of the **NewName** parameter, unless the path is identical to the path specified in
+the **Path** parameter. Otherwise, only a new name is permitted.
 
-This command attempts to rename the "project.txt" file in the current directory to "old-project.txt" in the "D:\Archive" directory. The result is the error shown in the output.
+This example attempts to rename the "project.txt" file in the current directory to "old-project.txt"

```

Comment:
In paragraph text, replace quotes with backticks. Same in Example 3 and 4.

Reply:
Fixed